docs/api-guide.md

```diff
@@ -696,28 +753,28 @@ Delete delete =
         .table("tbl")
         .partitionKey(partitionKey)
         .clusteringKey(clusteringKey)
-        .condition(condition)  // condition
+        .condition(condition) // condition
         .build();
 
-// Execute the Delete operation
+// Execute the `Delete` operation
 transaction.delete(delete);
 ```
 
 In addition to using the `deleteIf` condition, you can specify the `deleteIfExists` condition as follows:
 
 ```java
-// Build a deleteIfExists condition
+// Build a `deleteIfExists` condition
 MutationCondition deleteIfExistsCondition = ConditionBuilder.deleteIfExists();
 ```
 
 #### Mutate operation
 
-Mutate is an operation to execute multiple mutations (Put and Delete operations).
+Mutate is an operation to execute multiple mutations (`Put` and `Delete` operations).
```

Comment:
Should the part in parentheses say "for example"?

```suggestion
Mutate is an operation to execute multiple mutations (for example, `Put` and `Delete` operations).
```

Or, are there only two operations that can be used for mutations here?

Reply:
> Or, are there only two operations that can be used for mutations here?

I think so. Other operations are read operations (`Selection`) `Get` and `Scan`.

Reply:
> > Or, are there only two operations that can be used for mutations here?
> 
> I think so. Other operations are read operations (`Selection`) `Get` and `Scan`.

Got it. Thanks! I'll leave this part as is.